src/packages/excalidraw/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "excalidraw_karat",
-  "version": "0.0.19",
+  "version": "0.0.24",
```

Comment:
This has 24 committed but the [latest one is 25](https://www.npmjs.com/package/excalidraw_karat). I just want to double check that the commit matches the version that we will use in hyperpad.

Reply:
pushhhhhh

Reply:
sorry